AdminWebsite/AdminWebsite/Contracts/Responses/AllocationHearingsResponse.cs

```diff
@@ -25,15 +25,15 @@ public class AllocationHearingsResponse
     public string CaseNumber { get; set; }
     
     /// <summary>
-    /// The hearing case type
+    /// The hearing Service
```

Comment:
For this and the other backend changes I would probably just leave them as is (Case type), so that they still match the property and variable names. We're only updating the terminology for the user, we can still call it case type in the code

Reply:
done

Reply:
done